### DISCUSSION TITLE
HTML ListBoxes usability

### DISCUSSION THREAD

**Discussion by wqweto** [twinbasic/twinbasic#574] [General]
I just noticed that the all custom HTML ListBoxes in project Settings cannot locate entries by pressing item's first letter.

In `ALL AVAILABLE COM REFERENCES` I'm used to pressing "M" to jump to Microsoft's typelibs and then double click on the typelib to alternate the checkbox which in the HTML version of the ListBox is not working too.

Recent versions of Windows you can search in ListBox/Combo's Dropdown by typing start of the entry while older ones had jump-to by first letter only.

**Comment by WaynePhillipsEA** [comment]
Old habits die hard... this should now be much better in v0.13.50.
